admin-manual/maintenance/maintenance.rst

```diff
@@ -134,72 +142,280 @@ The index names are: `aips`, `aipfiles`, `transfers` and `transferfiles`.
 database by using the following command:
 
 .. code:: bash
 
-   mysqldump -u <your username> -p<your password> -c MCP > <filename of backup>
+   mysqldump -u <your username> -p <your password> -c MCP > <filename of backup>
+
+
+To restore from mysqldump file:
+
+.. code:: bash
+
+   mysql –user yourusername -p <your password> MCP < MCP_backup.sql
+
+Storage Service Database backup and restore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To backup the SQLite database and pointer files created by the storage service run:
+
+.. code:: bash
+
+  rsync -av /var/archivematica/storage_service /backup/location/storage_service
+  rsync -av /var/archivematica/storage-service/storage.db /backup/location/storage.db
+
+Note: The Storage Service must not be actively in use. Make sure the
+Storage Service is not running by stopping the nginx or storage-service
+services or by making the backup at a time that it is not in use.
+
+To restore Storage Service from backup:
+
+.. code:: bash
+
+  service archivematica-storage-service stop
+  rsync -av /backup/location/storage.db /var/archivematica/storage-service/storage.db
+  rsync -av /backup/location/storage_service /var/archivematica/storage_service
+  service archivematica-storage-service start
+
+Elasticsearch
+^^^^^^^^^^^^^
 
 Elasticsearch is used to store long-term data. Instructions and scripts for
 backing up and restoring Elasticsearch are available in the
 `Elasticsearch documentation`_.
 
+**Preconfiguration**
+
+The path.repo and snapshot repository have to be configured. For example, using
+/var/lib/elasticsearch/backup-repo as the repo path:
+
+.. code:: bash
+
+  mkdir /var/lib/elasticsearch/backup-repo
+  chmod 0755 /var/lib/elasticsearch/backup-repo
+  chown elasticsearch:elasticsearch /var/lib/elasticsearch/backup-repo
+
+Add this line to the /etc/elasticsearch/elasticsearch.yml file:
+
+.. code:: bash
+
+  path.repo: /var/lib/elasticsearch/backup-repo
+
+Restart elasticsearch: 
+
+.. code:: bash
+
+  service elasticsearch restart
+
+To use a new directory as snapshot repository, create and adjust permissions for one, like so:
+
+.. code:: bash
+
+  mkdir /var/lib/elasticsearch/backup-repo/es_backup_clientname
+  chmod 0755 /var/lib/elasticsearch/backup-repo/es_backup_clientname
+  chown elasticsearch:elasticsearch /var/lib/elasticsearch/backup-repo/es_backup_clientname
+
+Before any snapshot or restore operation can be performed, a snapshot repository
+should be registered in Elasticsearch. The repository settings are
+repository-type specific:
+
+.. code:: bash
+
+  curl -XPUT 'http://localhost:9200/_snapshot/es_backup_clientname' -d '{
+      "type": "fs",
+      "settings": {
+          "compress" : true,
+          "location": "/var/lib/elasticsearch/backup-repo/es_backup_clientname"
+      }
+  }'
+
+**Backing up Elasticsearch indexes**
+
+To make a backup (snapshot) for the aips and transfer indexes, a different name
```

Comment:
```suggestion
To make a backup (snapshot) for the AIP and transfer indexes, a different name
```

Reply:
I made the above suggestion but then I realized that maybe the index is literally called *aips*?

Reply:
I think it is, because of the example below, but I'd like someone else to confirm this~